pkg/apis/duck/v1alpha1/channel_defaulter.go

```diff
@@ -20,7 +20,7 @@ package v1alpha1
 //  Context: there is a cyclic dependency between eventing and messaging if we place this in messaging. Broker needs to
 //  depend on this, which is fine. But the problem arises due to messaging depending on eventing, mainly on
 //  Subscription-related objects for the Sequence type. We should first move Subscription down to messaging and then we
-//  can move this down. See https://github.com/knative/eventing/issues/1562.
+//  can move this down. See https://knative.dev/eventing/issues/1562.
```

Comment:
all issues link should still be under github

Reply:
THANKS! fixed.